### DISCUSSION TITLE
How do you find the currency from its code?

### DISCUSSION THREAD

**Discussion by antoineandrieu** [dinerojs/dinero.js#648] [Q&A] [answered]
I want to find the dinero currency in the `@dinero.js/currencies` package from its code. What is the simplest way to do it?

Like having a function `getCurrency` that takes a `string` as an input and returns the right dinero currency.

**Comment by antoineandrieu** [comment, marked as answer]
The simplest way is to import all the currencies and to find the right module from its name.

```
import * as currencies from '@dinero.js/currencies';

const currencyCode = 'USD';
const dineroCurrency = currencies[currencyCode];
```

**Reply by sarahdayan** [reply]
I recommend only importing the currencies that you know can be queried in your app, otherwise you'll bundle unused currencies.
